R/impact_central.R

```diff
@@ -182,8 +190,8 @@ impact_by_year_of_vaccination <- function(meta1, raw_impact, fvps, fvps_updates
 }
 
 determine_vaccine_delivery <- function(meta1){
-  setdiff(unlist(strsplit(meta1$vaccine_delivery[meta1$meta_type == "focal"], ",")),
-          unlist(strsplit(meta1$vaccine_delivery[meta1$meta_type=="baseline"], ",")))
+  setdiff(unlist(strsplit(as.character(meta1$vaccine_delivery[meta1$meta_type == "focal"]), ",")),
```

Comment:
Curious what corner case this works around

Reply:
Still curious here

Reply:
Still unsure - this was something Xiang added. Tests and vignettes still run successfully without it so have removed in a PR here https://github.com/vimc/vimpact/pull/16